doc/Developing.md

```diff
@@ -11,6 +10,30 @@ The following instructions are for Linux and macOS.
 
 For Windows instructions [click here](#building-on-windows).
 
+ * [**Prerequisites**](#prerequisites)
+ * [**Quick Start**](#quick-start)
+ * [**Clone the repository**](#clone-the-repository)
+ * [**Build core, extensions and examples packages**](#build-core-extensions-and-examples-packages)
+ * [**Run the browser-based example application**](#run-the-browser-based-example-application)
+ * [**Run the electron-based example application**](#run-the-electron-based-example-application)
+ * [**Rebuilding**](#rebuilding)
+ * [**Watching**](#watching)
+	 * [core and extension packages](#core-and-extension-packages)
```

Comment:
Just to be consistent: the core and the extension packages. Do you agree? If so, can we have that?